assets/js/index.js

```diff
@@ -44,4 +60,53 @@ function clearAddTodoInput() {
 function clearErrorMessage() {
   errorMessage.innerText = "";
   backendErrorMessage.innerText = "";
+}
+
+async function getTasks() {
+  try {
+    const response = await fetch(`${url}/todos`);
+    const { data } = await response.json();
+    return data;
+  } catch {
+     showNetworkError();
+  }
+}
+
+function formatDate(date) {
+  const dateTime = new Date(date);
+  const year = dateTime.toLocaleDateString("en-US", { year: "numeric" });
+  const month = dateTime.toLocaleDateString("en-US", { month: "short" });
+  const day = dateTime.toLocaleDateString("en-US", { day: "numeric" });
+
+  return `${day} ${month} ${year}`;
+}
+
+function showTasks(tasks) {
+  todoList.innerHTML = tasks.sort(
+    (a, b) => new Date(b.createdAt) -  new Date(a.createdAt)
+  ).map(({text, createdAt}) => `<li class="todo-item">
+    <input type="checkbox" />
+    <p class="todo"  title="${formatDate(createdAt)}">${text}</p>
+    <i class="fa-solid fa-pen-to-square edite-icon"></i>
+    <i class="fa-solid fa-xmark remove-icon"></i>
+    </li>`).join("");
+}
+
+async function filterItemAllTasks() {
+  const tasks = await getTasks();
+  filterItemAll.innerText = tasks.length;
+}
+
+async function filterItemCompletedTasks() {
```

Comment:
Function name is not very descriptive. Maybe something like: updateCompletedTasksCount

Reply:
Can apply the same thing to other functions.

Reply:
Done.